LondonBritishLibrary/orient/BLorient13278.xml

```diff
@@ -0,0 +1,232 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BLorient13278" xml:lang="en" type="mss">
```

Comment:
if you created the first version directly in Oxygen there was no need to create a copy, you could simply rename the file and update the `xml:id`

if you created the first version in the APP then it must be first saved in the `new `folder and then synced first